webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,45 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (conversationMetadata) {
+        if (
+            conversationMetadata != null &&
+            typeof conversationMetadata !== 'undefined'
+        ) {
+            var metadataToSend = Object.assign({}, conversationMetadata);
+            var metadataToShow = Object.assign({}, conversationMetadata.metadata);
```

Comment:
duplicate line

Reply:
both are cloning the different objects which are coming into the argument.